netconf.c

```diff
@@ -1032,7 +1032,7 @@ xpath_set_namespace (char *path, char **ns_href, char **ns_prefix, xmlDoc *doc,
     char *next;
     char *top_node;
 
-    if (!*ns_href || !*ns_prefix)
+    if (path && strlen (path) > 1 && (!*ns_href || !*ns_prefix))
```

Comment:
Why > 1? is "/" valid?

Reply:
This is a check to make sure the path is valid and more than a single character. The first character is usually a /. We are trying to get a first level node in the global instance document to match to the  name from the path and that requires a valid node name to look up, which must be at least 1 character.